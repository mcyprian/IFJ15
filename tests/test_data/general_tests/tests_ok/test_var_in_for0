double square(int i)
{
    return i*i;
}

int main()
{
    string s = "first\n";
    cout << s;

    for(int i = 0; i < 5; i = i + 1)
    {
        int d = square(i);
        string s = " in for\n";
        cout << d << s;
    }
    
    cout << s;

    return 42;
}

