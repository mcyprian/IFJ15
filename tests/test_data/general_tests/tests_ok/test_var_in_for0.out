first
0 in for
1 in for
4 in for
9 in for
16 in for
first
